<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stacked_widget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>651</width>
      <height>411</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="login_page">
     <widget class="QLabel" name="userLabel">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>160</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
     <widget class="QLabel" name="passLabel">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>190</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="userEdit">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>160</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="passEdit">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>190</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>230</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLabel" name="errorLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>210</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-style:italic; color:#ff0000;&quot;&gt;Error! - username or password is invalid!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="admin_page">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Admin</string>
      </property>
     </widget>
     <widget class="QPushButton" name="logout_button1">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>631</width>
        <height>381</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="inventory_tab_a">
       <attribute name="title">
        <string>Manage Inventory</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="membership_tab_a">
       <attribute name="title">
        <string>Membership</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="manager_page">
     <widget class="QPushButton" name="logout_button2">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="customer_purchase_t">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>39</y>
        <width>631</width>
        <height>381</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="inventory_tab_m">
       <attribute name="title">
        <string>Manage Inventory</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="membership_tab_m">
       <attribute name="title">
        <string>Membership</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Customer Purchase List</string>
       </attribute>
       <widget class="QTableView" name="tableView_Customer">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>11</y>
          <width>591</width>
          <height>351</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Revenue</string>
       </attribute>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>3</x>
          <y>5</y>
          <width>531</width>
          <height>341</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>546</x>
          <y>5</y>
          <width>69</width>
          <height>22</height>
         </rect>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>8/20/17</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8/21/17</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8/22/17</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8/23/17</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8/24/17</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8/25/17</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8/26/17</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Week 8/20</string>
         </property>
        </item>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>3</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Store Manager</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>672</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
